.eslintrc.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
   extends: '@callstack/eslint-config',
+  rules: {
+    "require-await": "error"
```

Comment:
good rule

Reply:
@thymikee imo it should be default in `@callstack/eslint-config`